### DISCUSSION TITLE
Dose it support chinese or janpanse?

### DISCUSSION THREAD

**Discussion by ElieenAndBella** [tecoholic/ner-annotator#90] [Q&A]
Maybe it is a bug.
as expected:
![image](https://github.com/tecoholic/ner-annotator/assets/54427695/5dfa2e6a-1d81-4f82-b45b-6869341dd69a)
not as expected:
![image](https://github.com/tecoholic/ner-annotator/assets/54427695/34747bd2-e8fb-446f-af74-ad387dcbd46a)

**Comment by alvi-khan** [comment]
I'm sorry you're facing this issue.

The app uses spaces between words to determine where a tag starts and ends, and I think the lack of spaces in Japanese and Chinese writing is causing the issue you're facing. There is a feature request for character-level annotation support (see here: #53) that should take care of this which I hope will be available soon.

**Reply by ElieenAndBella** [reply]
oh sad, maybe still take a long time

**Reply by alvi-khan** [reply]
On the contrary. The latest release now supports this. The website has also been updated.

[Windows/Linux Builds](https://github.com/tecoholic/ner-annotator/releases/latest)
[Website](https://tecoholic.github.io/ner-annotator/)

Please let us know if you face any bugs while using this!

**Reply by ElieenAndBella** [reply]
what the shit🤯 so fast! thank you!😍
